code/game/gamemodes/objective_items.dm

```diff
@@ -216,6 +216,16 @@
 /obj/item/gun/energy/e_gun/hos/add_stealing_item_objective()
 	return add_item_to_steal(src, /obj/item/gun/energy/e_gun/hos)
 
+/datum/objective_item/steal/compactshotty
```

Comment:
are you sure you wanna add this? this is just for antags like wizard or bb and its bulky, which makes it hard to store (this is we we buffed antique laser and hos gun to be normal sized)

Reply:
> are you sure you wanna add this? this is just for antags like wizard or bb and its bulky, which makes it hard to store (this is we we buffed antique laser and hos gun to be normal sized)

Maybe I'll change it to destory.